tests/test_search_issue.py

```diff
@@ -14,13 +16,20 @@ def setup_method(self):
         self.issue_page = IssuePage(self.driver)
         self.dashboard_page.open_dashboard_page()
 
+    @allure.tag("UI")
+    @allure.title('Find the valid issue via quick search')
     def test_quick_search_one_issue(self):
-        self.dashboard_page.quick_search(self.valid_issue_id)
-        assert self.issue_page.at_page()
+        with allure.step('Perform a quick search with valid issue id'):
+            self.dashboard_page.quick_search(self.valid_issue_id)
+            assert self.issue_page.at_page()
+        self.dashboard_page.attach_allure_screenshot("Target issue is found and opened.png")
```

Comment:
Please, move this code to pytest hook to avoid code duplication.
https://stackoverflow.com/questions/48099835/how-to-include-selenium-screenshot-in-pytest-bdd-for-passed-tests